app/src/main/res/drawable/stars.xml

```diff
@@ -0,0 +1,86 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think this file should be named background_login, not stars as it does not have any stars.